packages/app/features/settings/PushNotificationSettingsScreen.tsx

```diff
@@ -80,59 +84,67 @@ export function PushNotificationSettingsScreen({ navigation }: Props) {
   );
 
   return (
-    <View flex={1} backgroundColor="$background">
-      <ScreenHeader
-        title="Push Notifications"
-        backAction={() => navigation.goBack()}
-      />
-      <View marginTop="$m" marginHorizontal="$2xl" flex={1}>
-        <SizableText marginLeft="$m" marginTop="$xl" size="$m">
-          Configure what kinds of messages will send you notifications.
-        </SizableText>
+    <ChatOptionsProvider {...useChatSettingsNavigation()}>
```

Comment:
Why do we need ChatOptionsProvider here?

Reply:
The push notifications screen includes `GroupListItem` and `ChatListItem` both which I think call `useChatOptions` so it was crashing when opening (seems present on current develop too)